### DISCUSSION TITLE
Empty budget trigger/default budget for imported transactions

### DISCUSSION THREAD

**Discussion by fitzchivalrik** [firefly-iii/firefly-iii#9185] [Firefly III] [answered]
Hi,

when importing transactions, I would like to set a default budget - that is, every transaction, which was not assigned a budget by a rule, will get this budget assigned.
I thought I could do that with a rule at the bottom of a group which checks for "Is budget empty", but that seems not to be possible?
Is there another way to achieve something similar?

**Comment by JC5** [comment, marked as answer]
You could use the action "Has no budget"?

**Reply by fitzchivalrik** [reply]
Oh my gosh, thank you very much. I have tried to set it up a few times over the last weeks and always looked at the trigger list under "Budget XY", missing the other options which do not start with "Budget" completly.
Dunno what that says about me or the sorting of the list OTZ. Layer 8 problem, in my book.
